### PR TITLE
new data filtering guide for all langs

### PR DESCRIPTION
**Pull request by saolsen**

### CODE REVIEW

**Comment by gj** [comment]
Does this supersede #1422 ? Can we close that one?

**Comment by saolsen** [comment]
> Does this supersede #1422 ? Can we close that one?

no I don't think so, they probably have to be merged

File: docs/content/any/guides/data_filtering/index.md [hunk 1]
```diff
@@ -55,61 +38,49 @@ executed query.
...
-query that returns the union of the other two. For example if the two
-queries are SQL queries combine could `UNION` them. If they were HTTP
-requests `{{% exampleGet "combineQuery" %}}` could put them in an array and 
-could handle executing an array of queries and combining the results.
+`{{% exampleGet "executeQuery" %}}` takes a query and returns a list of the results.
 
-You can define functions that apply to all types with
-`{{% exampleGet "setDataFilteringQueryDefaults" %}}`. Or you can pass type
-specific ones when you register a class.
+You define the adapter to use with `{{% exampleGet "setDataFilteringAdapter" %}}`.
```
Comment (gw000) [inline docs/content/any/guides/data_filtering/index.md:83]: Maybe move this up under the adapter header so it's more prominent

**Review by gw000 — APPROVED** [review]
One small suggestion about making `set_data_filtering_adapter` a little more prominent, otherwise looks good!

**Pull request merged**